games/r.yaml

```diff
@@ -164,6 +164,24 @@
   video:
     youtube: KSoVdF9SvNY
 
+- name: REDriver2
+  type: remake
+  originals:
+  - Driver 2
+  repo: https://github.com/SoapyMan/REDRIVER2/
+  development: active
+  status: playable
+  lang:
+  - C++
+  framework:
+  - SDL
+  - SDL2
+  - OpenGL
+  license:
+  - Custom
```

Comment:
I can't find a license in their repo; if it's not specified then the license should be `As-is`